addons/project/static/src/views/project_task_kanban/project_task_kanban_record.js

```diff
@@ -1,24 +1,28 @@
 /* @odoo-module */
 
-import { Record } from '@web/views/relational_model';
+import { KanbanRecord } from "@web/views/kanban/kanban_record";
+import { useState } from "@odoo/owl";
+import { ProjectTaskKanbanCompiler } from "./project_task_kanban_compiler";
+import { SubtaskKanbanList } from "@project/components/subtask_kanban_list/subtask_kanban_list"
 
-export class ProjectTaskRecord extends Record {
-    async _applyChanges(changes) {
-        const value = changes.personal_stage_type_ids;
-        if (value && Array.isArray(value)) {
-            delete changes.personal_stage_type_ids;
-            changes.personal_stage_type_id = value;
-        }
-        await super._applyChanges(changes);
+export class ProjectTaskKanbanRecord extends KanbanRecord {
+    setup() {
+        super.setup();
+        this.state = useState({folded: true});
     }
 
-    get context() {
-        const context = super.context;
-        const value = context.default_personal_stage_type_ids;
-        if (value && Array.isArray(value)) {
-            context.default_personal_stage_type_id = value[0];
-            delete context.default_personal_stage_type_ids;
-        }
+    /**
+     * @override
+     */
+    get renderingContext() {
```

Comment:
Is it no longer needed, right? 🤔

Reply:
I believe we need it because we still use the ```state.folded``` condition in the compiler